app/src/main/java/co/devhack/compose/southamericanqualifiers/ui/activities/TeamsActivity.kt

```diff
@@ -0,0 +1,310 @@
+        Divider(
+            color = Color.Blue,
+            modifier = Modifier.padding(top = 8.dp, bottom = 8.dp)
+                .fillMaxWidth()
+        )
+        Text(
+            stringResource(R.string.teams),
+            modifier = Modifier
+                .align(Alignment.CenterHorizontally),
+            fontWeight = FontWeight.Bold
+        )
+        Teams(teams = teams)
+    }
+}
+
+@Composable
+fun HeaderQualifies(
+    modifier: Modifier = Modifier
+) {
+    Card(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(8.dp)
+            .height(100.dp),
+        elevation = 8.dp,
+        border = BorderStroke(2.dp, Color.Blue)
+    ) {
+        Row(horizontalArrangement = Arrangement.Center) {
+            Column(modifier = Modifier.align(alignment = Alignment.CenterVertically)) {
+                Text(
+                    text = stringResource(id = R.string.south_american_qualifiers)
+                        .toUpperCase(Locale.ROOT),
+                    fontWeight = FontWeight.Bold,
+                    style = MaterialTheme.typography.body2
+                )
+                Icon(
+                    Icons.Filled.SportsSoccer,
+                    tint = colorResource(id = R.color.purple_700)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun DetailMatchesQualifiers(
+    modifier: Modifier = Modifier,
+    teamsSize: Int,
+    playedMatches: Int,
+    totalMatches: Int,
+) {
+    Card(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(8.dp),
+        elevation = 8.dp,
+        border = BorderStroke(2.dp, Color.Blue)
+    ) {
+        Row(
+            Modifier.padding(16.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.SpaceEvenly
+        ) {
+            Column(
+                modifier = Modifier
+                    .padding(start = 8.dp)
+                    .align(Alignment.CenterVertically)
+            ) {
+                Text(stringResource(R.string.teams), fontWeight = FontWeight.Bold)
+                Text(
+                    text = teamsSize.toString(),
+                    modifier = Modifier.align(Alignment.CenterHorizontally)
+                )
+            }
+            Column(
+                modifier = Modifier
+                    .padding(start = 8.dp)
+                    .align(Alignment.CenterVertically)
+            ) {
+                Text(stringResource(R.string.match_played), fontWeight = FontWeight.Bold)
+                Text(
+                    text = "$playedMatches / $totalMatches",
+                    modifier = Modifier.align(Alignment.CenterHorizontally)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun Teams(
+    modifier: Modifier = Modifier,
+    teams: List<Team>
+) {
+    LazyColumnFor(items = teams,
+        modifier = modifier.fillMaxSize(),
+        itemContent = { team ->
+            Card(
```

Comment:
Can the items content be moved to its own Composable? We have too many nested components here.

Reply:
Ok , listo